codesign.py

```diff
@@ -474,41 +505,67 @@ def upload_zip_to_notary(archive_path):
             'Failed to upload file %s to the notary service' % archive_path)
 
 
-def check_status(uuid):
+def check_status(uuid, isNotaryTool):
     '''Check the status of our request'''
-    command = [
-        'xcrun',
-        'altool',
-        '--notarization-info',
-        uuid,
-        '-u',
-        CODESIGN_USERNAME,
-        '--password',
-        APP_SPECIFIC_PASSWORD,
-        ]
+    if isNotaryTool:
+        command = [
+            'xcrun',
+            'notarytool',
+            'info',
+            uuid,
+            '--password',
+            APP_SPECIFIC_PASSWORD,
+            '--apple-id',
+            CODESIGN_APPSTORE_ID,
+            '--team-id',
+            CODESIGN_TEAM_ID,
+            ]
+    else:
+        command = [
+            'xcrun',
+            'altool',
+            '--notarization-info',
+            uuid,
+            '-u',
+            CODESIGN_USERNAME,
+            '--password',
+            APP_SPECIFIC_PASSWORD,
+            ]
 
     log('Checking on the status of request: %s' % uuid)
     # Note that this tool outputs to STDOUT on Xcode 11, STDERR on earlier
     output = '\n'.join(run_and_return_output(command))
     log(output)
 
-    match = re.search('[ ]*Status: ([a-z ]+)', output)
+    if isNotaryTool:
+        match = re.search('[ ]*status: ([a-zA-z ]+)', output)
+    else:
+        match = re.search('[ ]*Status: ([a-z ]+)', output)
     if not match:
         log_and_exit('Unrecognized output from: %s' % ' '.join(command))
 
     status = match.group(1)
-    if status == 'success':
-        return True
-    if status == 'in progress':
-        log('Notarization is still pending...\n')
-        return False
+    if isNotaryTool:
+        if status == 'Accepted': #seems like accpeted is the equivalent of success, if it means in progress it never finishes
```

Comment:
seems like accpeted is the equivalent of success, if it means in progress it never finishes

Reply:
can manually verify these with `spctl`? https://stackoverflow.com/questions/55074168/how-do-i-verify-that-a-macos-pkg-is-notarized#:~:text=The%20notarization%20process%20works%20by,for%20it%20to%20be%20notarized.

Reply:
synced with Chris in microkitchen, was caused by some caching issues and was resolved.